jaeger-extension-tests/src/test/resources/bal/01_http_svc_test.bal

```diff
@@ -14,6 +14,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/io;
 import ballerina/http;
 import ballerina/observe; import ballerinax/jaeger as _;   // TODO: Remove extension module imports
```

Comment:
```suggestion
import ballerina/observe;
import ballerinax/jaeger as _;   // TODO: Remove extension module imports
```

Reply:
I kept it like this because we plan to remove this import soon and add it automatically. The tests depend on line numbers and we will need to change them again.